airbyte-commons-worker/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -55,31 +54,17 @@ public class DefaultNormalizationRunner implements NormalizationRunner {
 
   private Process process = null;
 
-  public enum DestinationType {
-    BIGQUERY,
-    MSSQL,
-    MYSQL,
-    ORACLE,
-    POSTGRES,
-    REDSHIFT,
-    SNOWFLAKE,
-    CLICKHOUSE,
-    TIDB
-  }
-
-  public DefaultNormalizationRunner(final DestinationType destinationType,
-                                    final ProcessFactory processFactory,
-                                    final String normalizationImageName) {
-    this.destinationType = destinationType;
-    this.processFactory = processFactory;
-    this.normalizationImageName = normalizationImageName;
-  }
-
   public DefaultNormalizationRunner(final ProcessFactory processFactory,
                                     final String normalizationImage) {
     this.processFactory = processFactory;
     this.normalizationImageName = normalizationImage;
-    this.destinationType = null;
+    this.destinationType = getDestinationTypeFromNormalizationImage(normalizationImage);
+  }
+
+  private String getDestinationTypeFromNormalizationImage(final String normalizationImage) {
+    String imageNameWithoutTag = normalizationImage.contains(":") ? normalizationImage.split(":")[0] : normalizationImage;
+    return "airbyte/normalization".equalsIgnoreCase(imageNameWithoutTag) ? "bigquery"
```

Comment:
Looking at the previous logic, it seems like this doesn't always mean "bigquery". Postgres seems to use the same base image:

https://github.com/airbytehq/airbyte/blob/b452d53e9730aa8bbcc1b1790180c74a38ff75ed/airbyte-commons-worker/src/main/java/io/airbyte/workers/normalization/NormalizationRunnerFactory.java#L35-L36

Reply:
@pedroslopez Then I see two ways to solve this issue:
1) I can parse the name of the docker image (not normalization) and take the definition type.
2) I can add an additional field to the `destination_definition` file (and also to the `actor_definition` table) and read the type from there

The second way seems more logical to me. What do you say to that? Maybe you see some other option how best to get this type?